src/Summoner/Project.hs

```diff
@@ -46,18 +43,9 @@ generateProject projectName Config{..} = do
     license  <- choose parseLicenseName "License: " $ ordNub (cLicense : universe)
 
     -- License creation
-    let licenseLink = "https://api.github.com/licenses/" <> githubLicenseQueryNames license
-    licenseJson <-
-      readProcess "curl"
-                  [ toString licenseLink
-                  , "-H"
-                  , "Accept: application/vnd.github.drax-preview+json"
-                  ]
-                  ""
+    fetchedLicense <- fetchLicense license
     year <- currentYear
-    let licenseText = case (decodeStrict $ pack licenseJson) :: Maybe License of
-            Just t  -> customizeLicense license (unLicense t) nm year
-            Nothing -> error "Broken predefined license list"
+    let licenseText = customizeLicense license (unLicense fetchedLicense) nm year
```

Comment:
:+1: on @vrom911 suggestion. I would really appreciate that `customizeLicense` function takes `License` and returns `License` as well.